tests/test_operators.cpp

```diff
@@ -1528,6 +1528,29 @@ TEST_CASE( "applyParamNamedPhaseFunc", "[operators]" ) {
                 applyReferenceOp(refVec, regs, totalNumQubits, diagMatr);
                 REQUIRE( areEqual(quregVec, refVec, 1E2*REAL_EPS) );
             }
+            SECTION( "SCALED_INVERSE_SHIFTED_NORM" ) {
+                
+                enum phaseFunc func = SCALED_INVERSE_SHIFTED_NORM;
+                int numParams = 2 + numRegs;
+                qreal* params = (qreal*) malloc(numParams * sizeof(*params));
```

Comment:
use var-length array (instead of malloc) to avoid memory leak on Catch2 error handling (which would also trigger LLVM address sanitisation failure)